docs/sources/flow/release-notes.md

```diff
@@ -41,6 +41,149 @@ This functionality is now only available in the main configuration.
 The `disable_high_cardinality_metrics` configuration argument is used by `otelcol.exporter` components such as `otelcol.exporter.otlp`.
 If you need to see high cardinality metrics containing labels such as IP addresses and port numbers, you now have to explicitly set `disable_high_cardinality_metrics` to `false`.
 
+### Breaking change: Label for children blocks of `prometheus.exporter.cloudwatch`, `prometheus.exporter.snmp` and `pyroscope.scrape` are removed.
+
+Previously in aforementioned components, there are children blocks that requires labels which would be used for naming. 
```

Comment:
```suggestion
Previously, the child blocks in the `prometheus.exporter.cloudwatch`, `prometheus.exporter.snmp`, and `pyroscope.scrape components required labels which would be used for naming. 
```

Was this all child blocks? Or just the ones in the table below?

Reply:
Only the ones in the table